modular_skyrat/modules/modular_ert/code/marine_override/modified_equipment.dm

```diff
@@ -0,0 +1,9 @@
+/datum/outfit/centcom/ert/marine
+	suit_store = /obj/item/gun/ballistic/automatic/dozer
```

Comment:
this gun ABSOLUTELY SUCKS, dont give em it

Reply:
:trollface: